addons/point_of_sale/static/src/app/screens/product_screen/product_list/control_panel/control_panel.xml

```diff
@@ -5,7 +5,7 @@
         <div class="products-widget-control" t-ref="products-widget-control">
             <div class="rightpane-header">
                 <t t-if="!props.hasNoCategories">
-                    <div class="categories-header">
+                    <div class="categories-header" t-attf-class="{{state.isMobile and 'scrollbar-hidden' or ''}}">
```

Comment:
```suggestion
                    <div class="categories-header" t-attf-class="{{state.isMobile ? 'scrollbar-hidden' : ''}}">
```
Can you apply this change if it works? Then we can merge.

Reply:
Done !